### DISCUSSION TITLE
How to reload YAML changes?

### DISCUSSION THREAD

**Discussion by patrickcollins12** [basnijholt/adaptive-lighting#284] [Q&A]
I've defined my adaptive lighting setup in YAML. What configuration section do you reload after changing adaptive lighting settings? Or do you just restart the whole server?

**Comment by th3w1zard1** [comment]
The only way I've found to reload adaptive lighting from the files in custom components is by restarting home assistant core.

**Reply by th3w1zard1** [reply]
I believe I've implemented this in #459 edit: this ended up not working

**Reply by jarrah31** [reply]
Thank you for trying @th3w1zard1 - can I just check if the ability to reload yaml configuration without a full HA restart is still not working please?
